docs/guides/using-ronin-db/index.md

```diff
@@ -158,16 +158,22 @@ there are pending migrations. You will have to run `ronin-db migrate`.
 
 <div class="level">
   <div class="level-left">
-    <a class="button" href="../using-the-ronin-cli/">
-      Using the Ronin CLI &raquo;
+    <a class="button" href="../using-ronin-repos/">
+      &laquo; Using Ronin Repos
```

Comment:
The "Using the Ronin DB" guide is supposed to come after the "Using the Ronin CLI" guide. The "Using the Ronin CLI" guide introduces the reader to the Ronin CLI, then other Ronin commands like `ronin-db` and `ronin-repos` are introduced.

Reply:
I understand but i was taking it in the sequence as mentioned in the Getting Started Page...So could you let me know the correct sequence you would wan to be implemented?

Reply:
Leave the linking sequence alone. I put some thought into the next guide that each guide should link to.

However, I think the order of the links under "Guides" should be changed. I will move "Using the Ronin Ruby Shell" down to "Ruby Quick Ref", since they are both related to using Ruby.

Reply:
Yes make sense that all 3 Ruby related link at the bottom together!